docs/builds/guides/integration/basic-api.md

```diff
@@ -30,9 +30,9 @@ Regardless of the chosen build, creating an editor is done using the static `cre
 Add an element that CKEditor should replace to your HTML page:
 
 ```html
-<textarea name="content" id="editor">
+<div name="content" id="editor">
 	&lt;p&gt;Here goes the initial content of the editor.&lt;/p&gt;
```

Comment:
Since you changed the tag type to a `div`  the content no longer should be encoded. Now it would show as a plain text. This issue is also visible once again later in this PR.